src/db/migrate.rs

```diff
@@ -269,6 +269,17 @@ fn migrate_inner(version: Option<Version>, conn: &Connection, apply_mode: ApplyM
             // downgrade query
             "DROP TABLE blacklisted_crates;"
         ),
+        migration!(
+            context,
+            // version
+            7,
+            // description
+            "Change sandbox_overrides's max_memory_bytes to BIGINT",
```

Comment:
Nit: Don't say what we're doing, say why.
```suggestion
            "Allow memory limits of more than 4 GB",
```
Other than this looks good :)

Reply:
Well, I kept it consistent with the other ones :sweat_smile: 

You're right though, updated.